src/Humanizer/Properties/Resources.is.resx

```diff
@@ -0,0 +1,540 @@
+  </data>
+    <data name="DateHumanize_MultipleDaysAgo_Plural" xml:space="preserve">
+    <value>fyrir {0} dögum</value>
+  </data>
+  <data name="DateHumanize_MultipleDaysAgo_Singular" xml:space="preserve">
+    <value>fyrir {0} degi</value>
+  </data>
+      <data name="DateHumanize_MultipleDaysFromNow_Plural" xml:space="preserve">
+    <value>eftir {0} daga</value>
+  </data>
+  <data name="DateHumanize_MultipleDaysFromNow_Singular" xml:space="preserve">
+    <value>eftir {0} dag</value>
+  </data>
+    <data name="DateHumanize_MultipleHoursAgo_Above20" xml:space="preserve">
+    <value>fyrir {0} klukkustundum</value>
+  </data>
+    <data name="DateHumanize_MultipleHoursAgo_Plural" xml:space="preserve">
+    <value>fyrir {0} klukkustundum</value>
+  </data>
+  <data name="DateHumanize_MultipleHoursAgo_Singular" xml:space="preserve">
+    <value>fyrir {0} klukkustund</value>
+  </data>
+    <data name="DateHumanize_MultipleHoursFromNow_Plural" xml:space="preserve">
+    <value>eftir {0} klukkustundir</value>
+  </data>
+  <data name="DateHumanize_MultipleHoursFromNow_Singular" xml:space="preserve">
+    <value>eftir {0} klukkustund</value>
+  </data>
+    <data name="DateHumanize_MultipleMinutesAgo_Above20" xml:space="preserve">
+    <value>fyrir {0} klukkustundum</value>
+  </data>
+    <data name="DateHumanize_MultipleMinutesAgo_Plural" xml:space="preserve">
+    <value>fyrir {0} mínútum</value>
+  </data>
+  <data name="DateHumanize_MultipleMinutesAgo_Singular" xml:space="preserve">
+    <value>fyrir {0} mínútu</value>
+  </data>
+      <data name="DateHumanize_MultipleMinutesFromNow_Singular" xml:space="preserve">
+    <value>eftir {0} mínútu</value>
+  </data>
+      <data name="DateHumanize_MultipleMonthsAgo_Plural" xml:space="preserve">
+    <value>fyrir {0} mánuðum</value>
+  </data>
+  <data name="DateHumanize_MultipleMonthsAgo_Singular" xml:space="preserve">
+    <value>fyrir {0} mánuði</value>
+  </data>
+      <data name="DateHumanize_MultipleMonthsFromNow_Plural" xml:space="preserve">
+    <value>eftir {0} mánuði</value>
+  </data>
+  <data name="DateHumanize_MultipleMonthsFromNow_Singular" xml:space="preserve">
+    <value>eftir {0} mánuð</value>
+  </data>
+    <data name="DateHumanize_MultipleSecondsAgo_Above20" xml:space="preserve">
+    <value>{0} seconds ago</value>
+  </data>
+  <data name="DateHumanize_MultipleSecondsAgo_Plural" xml:space="preserve">
+    <value>fyrir {0} sekúndu</value>
+  </data>
+  <data name="DateHumanize_MultipleSecondsAgo_Singular" xml:space="preserve">
+    <value>fyrir {0} sekúndu</value>
+  </data>
+    <data name="DateHumanize_MultipleSecondsFromNow_Plural" xml:space="preserve">
+    <value>eftir {0} sekúndur</value>
+  </data>
+  <data name="DateHumanize_MultipleSecondsFromNow_Singular" xml:space="preserve">
+    <value>eftir {0} sekúndu</value>
+  </data>
+    <data name="DateHumanize_MultipleYearsAgo_Above20" xml:space="preserve">
+    <value>{0} years ago</value>
```

Comment:
Is this correct?

Reply:
Thanks @clairernovotny, good catch. I originally wanted to trace their use and see if they would be caught in tests but then forgot to finish cleaning up. I have translated them, and pushed a new commit.